wordpress-mobile/WordPress-iOS
# UL&S: bug fix: make the TextLinkButtonTableViewCell button word wrap, not truncate

Ref #14009 
Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/310

### To test:
1. Set the `.unifiedSiteAddress` feature flag to `BuildConfiguration.current == .localDeveloper`
2. `rake dependencies`
3. Build and run
4. Set dynamic text to max: Settings app > Accessibility > Display & Text Size > Larger Text > Switch on Larger Accessibility Sizes, then pull the bottom slider to max font size.
4. Log out if logged in
5. Log In > Or log in by entering your site address
6. Expected: "Find your site address" word-wraps instead of truncating.

**Before:**
<a href="https://user-images.githubusercontent.com/1816888/85751205-9acb8e00-b6c7-11ea-9cb0-2bea5f1c6ef7.png"><img src="https://user-images.githubusercontent.com/1816888/85751205-9acb8e00-b6c7-11ea-9cb0-2bea5f1c6ef7.png" width="350" /></a>

**After:**
<a href="https://user-images.githubusercontent.com/1062444/85781913-d11b0480-b6eb-11ea-9d3d-194a0c1d459b.png"><img src="https://user-images.githubusercontent.com/1062444/85781913-d11b0480-b6eb-11ea-9d3d-194a0c1d459b.png" width="350" /></a>

PR submission checklist:
